src/lib.rs

```diff
@@ -1196,17 +1196,33 @@ pub fn anyref_heap_live_count() -> u32 {
 pub trait UnwrapThrowExt<T>: Sized {
     /// Unwrap this `Option` or `Result`, but instead of panicking on failure,
     /// throw an exception to JavaScript.
+    #[cfg(all(feature = "std", debug_assertions))]
+    #[track_caller]
+    fn unwrap_throw(self) -> T {
```

Comment:
Instead of a duplicate method could this be inlined into the same function with something like:

```rust
if cfg!(debug_assertions) {
    // this method
} else {
    // existing method
}
```